src/js/variant-widget/eva-variant-widget-panel.js

```diff
@@ -257,7 +257,9 @@ EvaVariantWidgetPanel.prototype = {
                _this.formPanelVariantFilter.panel.getForm().findField('region').setValue('KB665398:1-200000');
             }else if (e.species == 'sratti_ed321v504') {
                 _this.formPanelVariantFilter.panel.getForm().findField('region').setValue('SRAE_chr2:10000-20000');
-            } else {
+            }else if (e.species == 'hsapiens_grch37') {
+                _this.formPanelVariantFilter.panel.getForm().findField('region').setValue('2:48000000-49000000');
+            }else {
```

Comment:
Using a switch (instead of multiple if-else) to map species to value, then calling `setValue` only at the end, would simplify this code block.